content/400-reference/300-database-reference/01-database-features.mdx

```diff
@@ -34,12 +34,14 @@ This section describes which database features exist on the databases that are c
 
 | Deletion behavior | PostgreSQL | SQL Server | MySQL | SQLite | Prisma schema | Prisma Client | Prisma Migrate |
 | ----------------- | ---------- | ---------- | ----- | ------ | ------------- | ------------- | -------------- |
-| `CASCADE`         | ✔️         | ✔️         | ✔️    | ✔️     | Not yet       | ✔️            | Not yet        |
+| `CASCADE`         | ✔️         | ✔️         | ✔️    | ✔️     | **✔️**†        | ✔️            | Not yet        |
```

Comment:
I don't understand this change.
Why only this line?

With 2.26.0 and referentialActions flag all of these should be possible in Prisma schema? and also working with Migrate.

Reply:
Oh of course! My mind was focused on cascade deletes 🤦‍♂️ Have pushed a fix